docs/source/getting_started.rst

```diff
@@ -0,0 +1,188 @@
+Getting Started
+============
+
+
+Glossary
+--------
+
+- A ``DataSource`` represents a way to retrieve data from database. Typically, this corresponds to a table in the database. Yet, it could also be a more elaborate object. See the section on 'Alternative ``DataSource`` s' for more detail.
+
+- A ``Constraint`` captures a concrete expectation between either two ``DataSource`` s or a single ``DataSource`` and a reference value.
+
+- A ``Requirement`` captures all ``Constraint`` s between two given ``DataSource`` s or all ``Constraint`` s within a single ``DataSource``. If a ``Requirement`` refers links to two ``DataSource`` s, it is a ``BetweenRequirement``. If a ``Requirement`` merely refers to a single ``DataSource``, it is a ``WithinRequirement``.
+
+- Conceptually, a 'specification' captures all ``Requirement`` s against a database. In practice that means it is usually a separate python file which:
+  
+  - gathers all relevant ``Requirement`` s
+  - turns these ``Requirement`` s' ``Constraint`` s into individual tests
+  - can be 'tested' by pytest
+
+
+Creating a specification
+------------------------
+
+In order to get going, you might want to use the following snippet in a new python file.
+This file will represent a specification.
+
+.. code-block:: python
+
+    import pytest
+    import sqlalchemy as sa
+    from dbcheck.pytest_integration import collect_data_tests
+
+
+    @pytest.fixture(scope="module")
+    def datajudge_engine():
+        # TODO: Adapt connection string to database at hand.
+        return sa.create_engine("your_connection_string")
+
+
+    # TODO: Insert Requirement objects to list.
+    test = []
+
+    requirements_constraint = collect_data_tests(requirements)
+
+This file will eventually lead as an input to pytest. More on that in the section 'Testing a specification'.
+
+In case you haven't worked with sqlalchemy engines before, you might need to install drivers to connect to your database. You might want to install snowflake-sqlalchemy when using Snowflake, pyscopg when using Postgres and platform-specific drivers (`Windows <https://docs.microsoft.com/en-us/sql/connect/odbc/windows/microsoft-odbc-driver-for-sql-server-on-windows?view=sql-server-ver15>`_, `Linux <https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-ver15>`_, `macOS` <https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos?view=sql-server-ver15>`_) when using MSSQL.
```

Comment:
```suggestion
In case you haven't worked with sqlalchemy engines before, you might need to install drivers to connect to your database. You might want to install snowflake-sqlalchemy when using Snowflake, pyscopg when using Postgres and platform-specific drivers (`Windows <https://docs.microsoft.com/en-us/sql/connect/odbc/windows/microsoft-odbc-driver-for-sql-server-on-windows?view=sql-server-ver15>`_, `Linux <https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-ver15>`_, `macOS <https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos?view=sql-server-ver15>`_) when using MSSQL.
```